plugins/k8sauditeks/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2022 The Falco Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+SHELL=/bin/bash -o pipefail
+GO ?= go
+CURL ?= curl
+
+FALCOSECURITY_FALCO_REPO ?= falcosecurity/falco
+FALCOSECURITY_FALCO_REVISION ?= 65435d4418638176f68df9017df36cd5d94ba083
+
+TEST_FILES_DIR := test_files
+NAME := k8sauditeks
```

Comment:
I believe a better name for this pluginnwould be `eks-k8saudit`. It's more readable and it sorts of hints that this is a "flavor" of k8saudit. It's also in line with the current sample docs where this was first envisioned and mentioned: https://falco.org/docs/plugins/#plugin-event-sources-and-interoperability

Reply:
I prefer `k8saudit-eks`, it allows to have all `k8saudit-X` plugins listed side by side in the repository, wdyt?

Reply:
Yeah that works for me!